### PR TITLE
Added config for deprecated 'HorizontalPodAutoscaler' apiversion

### LINKED ISSUE #67: Add deprecated "HorizontalPodAutoscaler" apiversion
Using this plugin and I noticed that there is no specification for the `HorizontalPodAutoscaler` resource as specified in the [Deprecated API Migration Guide](https://kubernetes.io/docs/reference/using-api/deprecation-guide/). I would love to create a PR for this.

### PR DESCRIPTION
**Pull request by hydeenoble**
deprecated "HorizontalPodAutoscaler" apiversion. 

The API Version is changed due to deprecated apiversion.
https://kubernetes.io/docs/reference/using-api/deprecation-guide/

Fixes #67

### CODE REVIEW

**Review by hickeyma — CHANGES_REQUESTED** [review]
@hydeenoble  It looks like you didn't rebase with main. It seems you have pulled in pervious changes into this PR.

**Comment by hydeenoble** [comment]
> @hydeenoble It looks like you didn't rebase with main. It seems you have pulled in pervious changes into this PR.

Yeah, that's what I did. What command should I run instead?

**Comment by hickeyma** [comment]
@hydeenoble You need to rebase your branch with the latest main branch.

**Comment by hydeenoble** [comment]
> @hydeenoble You need to rebase your branch with the latest main branch.

Done! @hickeyma

File: config/Map.yaml [hunk 1]
```diff
@@ -187,6 +187,9 @@ mappings:
     newAPI: "apiVersion: storage.k8s.io/v1\nkind: VolumeAttachment\n"
     deprecatedInVersion: "v1.13"
     removedInVersion: "v1.22"
+  - deprecatedAPI: "apiVersion: autoscaling/v2beta1\nkind: HorizontalPodAutoscaler\n"
+    newAPI: "apiVersion: autoscaling/v2\nkind: HorizontalPodAutoscaler\n"
+    deprecatedInVersion: "v1.23"
```
Comment (hickeyma) [inline config/Map.yaml:192]: Missing `removedInVersion` property

**Review by hickeyma — CHANGES_REQUESTED** [review]

File: config/Map.yaml (hunk 1 above)
Comment (hydeenoble) [reply config/Map.yaml:192]: Done!

**Review by hickeyma — CHANGES_REQUESTED** [review]
`.DS_Store` should not be part of the commit.

**Comment by hydeenoble** [comment]
> `.DS_Store` should not be part of the commit.

Removed!. That was an oversight.

**Review by hickeyma — APPROVED** [review]
LGTM, thanks @hydeenoble

**Pull request merged**